src/solver.jl

```diff
@@ -9,79 +10,106 @@
 
 
 # Policy struct created according to one used in DiscreteValueIteration
+
+#TODO: Dosctring
+struct FiniteHorizonSolver <: Solver
```

Comment:
It would be beneficial to add constructor function that takes the parameter as a keyword.